ldap/src/test/java/pl/edu/icm/unity/ldap/LdapNodeUtilsTest.java

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
This file should be removed - or better filled with tests and proper license header

Reply:
+1